File: count_to_1Billion.cc
#include <stdio.h>
#include <time.h>

int main(){
    printf("Counting to 1 billion\n");
    int n = 0;
    float t;
    clock_t c;
    c = clock();

    while(n<1000000000){
        n++;
    }

    c = clock() - c;
    t = (float)c/CLOCKS_PER_SEC;
    printf("%d",n);
    printf("\n Elapsed time: %f",t);
}

File: count_to_1Billion.py
import time

# Start timing
start_time = time.time()

# Loop to count to 1 billion
for i in range(1, 1000000001):
    pass

# End timing
end_time = time.time()

# Calculate elapsed time
elapsed_time = end_time - start_time
print(elapsed_time)
